core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -264,3 +265,12 @@ class ClientMetricsConfigHandler(private val clientMetricsManager: ClientMetrics
     clientMetricsManager.updateSubscription(subscriptionGroupId, properties)
   }
 }
+
+/**
+ * The GroupConfigHandler will process individual group config changes in ZK.
+ */
+class ConsumerGroupConfigHandler(private val consumerGroupConfigManager: ConsumerGroupConfigManager) extends ConfigHandler with Logging {
```

Comment:
`GroupConfigHandler` like the comment please :)

Reply:
Done